kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -42,11 +42,11 @@ data:
           # When the setting is set to false, only the kube events with !normal event type will be collected
           enabled = false
           # When this is enabled (enabled = true), all kube events including normal events will be collected
-       #[log_collection_settings.schema]
+       [log_collection_settings.schema]
           # In the absence of this configmap, default value for containerlog_schema_version is "v1"
           # Supported values for this setting are "v1","v2"
           # See documentation at https://aka.ms/ContainerLogv2 for benefits of v2 schema over v1 schema before opting for "v2" schema
-          # containerlog_schema_version = "v2"
+          containerlog_schema_version = "v2"
        #[log_collection_settings.enable_multiline_logs]
```

Comment:
Should we also mention here that v2 is needed for multiline?

Reply:
our public docs already mention this and there are v2 usecases outside of multiline.

Reply:
No I meant should we mention in multiline that v2 is needed ?

Reply:
added to multiline section